po/ja-kana.po

```diff
@@ -0,0 +1,5137 @@
+#: js/basicblocks.js:2873
+#: js/basicblocks.js:2883
+#: js/basicblocks.js:2915
+#: js/basicblocks.js:2960
+#: js/basicblocks.js:2964
+#: js/basicblocks.js:2970
+#: js/macros.js:44
+#: js/blocks.js:784
+#: js/blocks.js:2385
+#: js/blocks.js:2386
+#: js/blocks.js:2578
+#: js/blocks.js:2784
+#: js/blocks.js:2940
+#: js/blocks.js:3503
+#: js/blocks.js:4308
+#: js/block.js:720
+#: js/block.js:3892
+#: js/block.js:3901
+#: js/block.js:3912
+#: js/activity.js:2732
+#: js/analytics.js:475
+#: js/palette.js:681
+#: js/palette.js:690
+#: js/palette.js:699
+#: js/palette.js:708
+#: js/palette.js:1983
+#: js/palette.js:1994
+#: js/palette.js:2005
+#: js/palette.js:2016
+#: js/turtledefs.js:23
+#: js/basicblocks.js:1148
+#: js/basicblocks.js:1159
+#: js/basicblocks.js:1170
+#: js/basicblocks.js:2817
+#: js/basicblocks.js:2848
+#: js/basicblocks.js:2855
+#: js/basicblocks.js:2884
+#: js/basicblocks.js:2894
+#: js/basicblocks.js:2926
+#: js/basicblocks.js:2971
+#: js/basicblocks.js:2975
+#: js/basicblocks.js:2981
+#: js/macros.js:44
+#: js/blocks.js:784
+#: js/blocks.js:2385
+#: js/blocks.js:2386
+#: js/blocks.js:2578
+#: js/blocks.js:2784
+#: js/blocks.js:2940
+#: js/blocks.js:3514
+#: js/blocks.js:4319
+#: js/block.js:720
+#: js/block.js:3892
+#: js/block.js:3901
+#: js/block.js:3912
+#: js/activity.js:2732
+#: js/analytics.js:475
+#: js/palette.js:681
+#: js/palette.js:690
+#: js/palette.js:699
+#: js/palette.js:708
+#: js/palette.js:1983
+#: js/palette.js:1994
+#: js/palette.js:2005
+#: js/palette.js:2016
+#: js/turtledefs.js:23
+#: js/basicblocks.js:1148
+#: js/basicblocks.js:1159
+#: js/basicblocks.js:1170
+#: js/basicblocks.js:2817
+#: js/basicblocks.js:2848
+#: js/basicblocks.js:2855
+#: js/basicblocks.js:2884
+#: js/basicblocks.js:2894
+#: js/basicblocks.js:2926
+#: js/basicblocks.js:2971
+#: js/basicblocks.js:2975
+#: js/basicblocks.js:2981
+#: js/macros.js:44
+#: js/blocks.js:784
+#: js/blocks.js:2385
+#: js/blocks.js:2386
+#: js/blocks.js:2578
+#: js/blocks.js:2784
+#: js/blocks.js:2940
+#: js/blocks.js:3514
+#: js/blocks.js:4319
+#.TRANS: a stack of blocks to run (an action to take)
+msgid "action"
+msgstr "アクション"
+
+#: js/block.js:723
+#: js/logo.js:1347
+#: js/logo.js:10186
+#: js/turtle.js:567
+#: js/turtle.js:568
+#: js/basicblocks.js:2963
+#: js/lilypond.js:586
+#: js/lilypond.js:737
+#: js/lilypond.js:775
+msgid "start"
+msgstr "スタート"
+
+#: js/block.js:726
+msgid "matrix"
+msgstr "ピッチ・じきぎょうれつ"
+
+#: js/block.js:729
+#: js/basicblocks.js:837
+#: plugins/rodi.rtp:328
+## is this for "status widget"? Maybe ステータスモニター;
+## Also 現状 http://jisho.org/word/%E7%8F%BE%E7%8A%B6
+msgid "status"
+msgstr "ステータス"
+
+#: js/block.js:732
+#: js/block.js:753
+#: js/palette.js:729
+#: js/turtle.js:409
+#: js/turtledefs.js:23
+#: js/basicblocks.js:996
+#: js/basicblocks.js:2199
+#: js/lilypond.js:569
+#: js/lilypond.js:727
+#: js/lilypond.js:765
+msgid "drum"
+msgstr "ドラム"
+
+#: js/block.js:735
+## ひらがな＝じょうぎ
+msgid "ruler"
+msgstr "ルーラー"
```

Comment:
Not changing because this is the short name for "rhythm ruler" and everyone seemed confused by the other name I chose.

I will delete my note from the file which is probably confusing as it was my note before the group expressed preference in the translation リズム・ルーラー.

BTW, you probably already know this but you can go to the file cited (in this case js/block.js, line 735) to get an idea of the meaning.